React/microstates/src/index.js

```diff
@@ -0,0 +1,45 @@
+/************************************************
+ * This example is one of many examples
+ *         -= The React State Museum =-
+ * View other state management options and read
+ * more in the blog post and the master repo:
+ *
+ * https://github.com/GantMan/ReactStateMuseum
+ ************************************************/
+import React, { Component } from "react";
+import { render } from "react-dom";
+import ListItems from "./Components/listItems";
+import AddItems from "./Components/addItem";
+import State from "@microstates/react";
+import List from "./list";
+
+const styles = {
+  fontFamily: "sans-serif",
+  textAlign: "center"
+};
+
+export default class App extends Component {
+  render() {
+    return (
+      <State
+        type={List}
+        value={{ allItems: ["nachos", "burritos", "hot dog"] }}
+      >
+        {list => (
+          <div style={styles}>
+            <h2>Welcome to Microstates</h2>
+            <ListItems allItems={list.state.allItems} />
+            <AddItems
```

Comment:
Can you please switch these two?  The AddItems is usually above the ListItems in all other examples.

![image](https://user-images.githubusercontent.com/997157/39820166-d2175bf4-536a-11e8-871f-e1c8b50c5450.png)

Reply:
Ha! I didn't even realize I flipped them around. I'll get a change pushed up for sure